### ISSUE TITLE
Gauntlet Color Contrast Issue

### ISSUE THREAD

**Issue by AricCougar** [IAmPicard/StarTrekTimelinesSpreadsheet#179]
**Describe the bug**
The NEW Gauntlet upgrade presents 2 color contrast issues.  The light green circles with white text on the characters images cannot be made out on some systems.  It looks washed out, but can be fixed with higher contrast of colors, perhaps changing the text in the light green circle from white to black, or change the light green circle to dark green and keep the white text.

The other issue is light gray text on white background located just under the rank box.  It cant be read either, due to lack of color contrast, but can be fixed if changing the light grey text to black.

**To Reproduce**
1. Go to Gauntlet Tab in STTCM v0.8.0 when there are playable options.
2. Scroll down to the opponent match up boxes, and look at the new % circles at the feet of the character images.  Notice the white on light green is hard to read the text.
4. Then click to engage a battle
5. Scroll up to the Your Rank Box, and look directly below it at the light grey and light green text on white background.  Notice it hard to make out the text due to the color contrast. 

**Expected behavior**
The text in these 2 locations should have enough color contrast to be readable on all devices of all sizes with reasonable normal brightness.

**Screenshots**
See 2 screen shots from my desktop version today. 
![color contrast issue 2 gauntlet](https://user-images.githubusercontent.com/42733197/46247884-1d2dfd80-c3cf-11e8-9290-83e7d43a8a86.GIF)
![color contrast issue 1 gauntlet](https://user-images.githubusercontent.com/42733197/46247885-1dc69400-c3cf-11e8-8e99-2a8627aae7b0.GIF)

**Additional context**

**Comment by IAmPicard** [comment]
Thank you for the detailed report. FWIW, I tent to use it mostly in light mode (white background), and these issues are less prevalent there.

**Comment by AricCougar** [comment]
Ah, that makes sense.  I just switched to see it your way, and you are right in light mode, there is NO issue with the box under Rank, it looks great. However I still can't make out the % on the crew members below.  Perhaps Dark mode could be modified to adjust these 2 colors to work for it.  It will be interesting to see what solution you come up with in the end.


One question.... have you ever considered changing the color circles for the %, based on if its 5% 25% 45% 65% (perhaps green, yellow, orange, red) or something like that?   Since you already autocalculate the % Chance already, another visual to help select a match up, may be less useful.  But i thought id still ask your opinion on that.


Glad to help with the report.  I'll keep a watch for other things you may want reported.


________________________________
